css/properties/touch-action.json

```diff
@@ -0,0 +1,215 @@
+{
+  "css": {
+    "properties": {
+      "touch-action": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/touch-action",
+          "support": {
+            "webview_android": {
+              "version_added": "36"
+            },
+            "chrome": {
+              "version_added": "36"
+            },
+            "chrome_android": {
+              "version_added": "36"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "52",
+                "notes": "Not applicable to Firefox platforms that support neither pointer nor touch events."
+              },
+              {
+                "version_added": "29",
+                "version_removed": "52",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.touch_action.enabled",
+                  "value_to_set": "true"
+                },
+                "notes": "Not applicable to Firefox platforms that support neither pointer nor touch events."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "29",
+                "version_removed": "52",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.touch_action.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": [
+              {
+                "version_added": "11",
+                "notes": "IE additionally supports the non-standard values <a href='https://msdn.microsoft.com/zh-cn/library/windows/apps/hh767313.aspx'>double-tap-zoom, cross-slide-x and cross-slide-y</a>."
+              },
+              {
+                "version_added": "10",
+                "prefix": "-ms-"
+              }
+            ],
+            "ie_mobile": [
+              {
+                "version_added": "11",
+                "notes": "IE additionally supports the non-standard values <a href='https://msdn.microsoft.com/library/Hh772044'>double-tap-zoom, cross-slide-x and cross-slide-y</a>."
+              },
+              {
+                "version_added": "10",
+                "prefix": "-ms-"
+              }
+            ],
+            "opera": {
+              "version_added": "23"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": false,
+              "notes": "See <a href='https://bugs.webkit.org/show_bug.cgi?id=133112'>WebKit bug 133112</a>."
+            },
+            "safari_ios": {
+              "version_added": "9.1",
+              "partial_implementation": true,
```

Comment:
This is an interesting case.
I think we should drop "partial_implementation" and the iOS "notes" and replace them by listing all the value link you did for pinch-zoom and unidirectional-pan. That way each value will have its machine-readable compat data.

The experiment pan-up, pan-down, pan-left, pan-right values also have specific values, so this will solve also this.

Reply:
Just for context, at the moment the rows correlate to level 1 and level 2 values, plus `pinch-zoom` that was originally non-standard but got added to WHATWG Compatibility Standard (charter issues prevented it from getting directly included in L2). Aside from `manipulation`'s dual purpose of overcoming the 300ms click delay, it's difficult to imagine an engine implementing different the L1 or highly correlated L2 values across different versions, producing 9+ table rows.